#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_rviz_plugins'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()
gen.add("use_parent_link", bool_t, 0, "", False)
gen.add("marker_scale", double_t, 0, "", 1.02, 0.9, 1.1)
gen.add("on_red", double_t, 0, "", 1.0, 0, 1.0)
gen.add("on_green", double_t, 0, "", 0.0, 0, 1.0)
gen.add("on_blue", double_t, 0, "", 0.0, 0, 1.0)
gen.add("on_alpha", double_t, 0, "", 0.8, 0, 1.0)
gen.add("visualize_off", bool_t, 0, "", False)
gen.add("off_red", double_t, 0, "", 0.0, 0, 1.0)
gen.add("off_green", double_t, 0, "", 0.0, 0, 1.0)
gen.add("off_blue", double_t, 0, "", 1.0, 0, 1.0)
gen.add("off_alpha", double_t, 0, "", 0.8, 0, 1.0)

exit (gen.generate (PACKAGE, "jsk_rviz_plugins", "ContactStateMarker"))
